src/main/webapp/js/components/api/fetcher.js

```diff
@@ -0,0 +1,134 @@
+import apiClient from './api-client';
+
+const BASE_URL = '/api';
+const VACCINEES = `${BASE_URL}/vaccinee`;
+const CAMPAIGN_MESSAGE  = `${BASE_URL}/campaignMessage`;
+const ENROLLMENT_GROUP = `${BASE_URL}/enrollmentGroup`;
+const FIELD_CONFIG = `${BASE_URL}/fieldConfig`;
+const KEY_COMMUNITY_PERSON = `${BASE_URL}/keyCommunityPerson`;
+const LANGUAGE = `${BASE_URL}/language`;
+const SITE = `${BASE_URL}/site`;
+const VISIT = `${BASE_URL}/visit`;
+const VISIT_TYPE = `${BASE_URL}/visitType`;
+
```

Comment:
I'm not sure if this file is needed, I think all of this will be part of the reducers